PushNotifications.md

```diff
@@ -328,6 +329,13 @@ Users can listen to messages in each platform using the native message listeners
   }
 ```
 
+Then, in your Android Manifest, disable Ably Flutter's broadcast receiver by removing it from your Manifest using `tools:node="remove"`. This will prevent Ably from listening for new push notifications and launching your Flutter application. You can do this by adding the following between `<application>` and `</application>`:
+```xml
+<receiver android:name="io.ably.flutter.plugin.push.FirebaseMessagingReceiver"
+    tools:node="remove">
+</receiver>
+```
```

Comment:
Nice explanation 👍

Reply:
Nothing to action here, as far as I can see, so I'm resolving this conversation on @ikbalkaya's behalf.